save-backend/src/main/kotlin/com/saveourtool/save/backend/service/OrganizationService.kt

```diff
@@ -69,7 +69,18 @@ class OrganizationService(
      * @param organization
      * @return organization
      */
-    fun updateOrganization(organization: Organization): Organization = organizationRepository.save(organization)
+    @Suppress("UnsafeCallOnNullableType")
+    fun updateOrganization(organization: Organization): Organization = organizationRepository.findByName(
```

Comment:
This is a duplicate of your logic from the Controller. She shouldn't be here at all. The service should not contain any additional logic and should only call the Save method.

Reply:
Agree, forgot to remove it

Reply:
Done